website/highlights/2020-07-09-add-musl-and-glibc-support-to-install-sh.md

```diff
@@ -0,0 +1,52 @@
+---
+last_modified_on: "2020-07-14"
+$schema: "/.meta/.schemas/highlights.json"
+title: "glibc enhancements"
+description: "If your Linux uses glibc, Vector will too."
+author_github: "https://github.com/hoverbear"
+hide_on_release_notes: false
+pr_numbers: [2969,2518]
+release: "0.10.0"
+tags: ["type: breaking change", "domain: operations"]
```

Comment:
I don't think we should tag this as a `type: breaking change` since it is not intended to be. I'd rather this be a `type: performance`.

Reply:
This is absolutely a breaking change! It changes how we link!